pkgs/development/libraries/sundials/3.x.nix

```diff
@@ -0,0 +1,29 @@
+{ cmake, fetchurl, python, stdenv }:
+
+stdenv.mkDerivation rec {
+
+  pname = "sundials";
+  version = "3.2.1";
+
+  src = fetchurl {
+    url = "https://computation.llnl.gov/projects/${pname}/download/${pname}-${version}.tar.gz";
+    sha256 = "0238r1qnwqz13wcjzfsbcfi8rfnlxcjjmxq2vpf2qf5jgablvna7";
+  };
+
+
+  cmakeFlags = [
+    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples"
+  ];
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ python ];
+
+  meta = with stdenv.lib; {
+    description = "Suite of nonlinear differential/algebraic equation solvers";
+    homepage    = https://computation.llnl.gov/projects/sundials;
+    platforms   = platforms.all;
+    maintainers = [ maintainers.idontgetoutmuch ];
```

Comment:
Did you ask @idontgetoutmuch ? I think you should add yourself considering this is only used in a package you're adding.

Reply:
Just saying because not everyone is interested in maintaining multiple versions when they're not using those versions themselves.

Reply:
Yeah, we were both working on getting scikits-odes (and its dependencies) properly into nixpkgs, to make it more accessible, and had some off-github discussion.

Also see https://github.com/bmcage/odes/issues/98 for being able to move to the recent sundials version ;-)

Reply:
I am happing to be maintainer.